lib/fluent/plugin/buffer.rb

```diff
@@ -55,7 +55,11 @@ class BufferChunkOverflowError < BufferError; end # A record size is larger than
       # if chunk size (or records) is 95% or more after #write, then that chunk will be enqueued
       config_param :chunk_full_threshold, :float, default: DEFAULT_CHUNK_FULL_THRESHOLD
 
-      Metadata = Struct.new(:timekey, :tag, :variables)
+      Metadata = Struct.new(:timekey, :tag, :variables) do
+        def nil?
```

Comment:
I changed this method name `empty?`
https://github.com/fluent/fluentd/pull/1154/files#diff-7d794f77abc9222413b9120533cff327R59